modules/vpc/variables.tf

```diff
@@ -89,3 +89,21 @@ variable "network_profile" {
       * projects/{projectId}/global/networkProfiles/{network_profile_name}
     EOT
 }
+
+variable "bgp_always_compare_med" {
+  type        = bool
+  description = "If set to true, the Cloud Router will use MED values from the peer even if the AS paths differ. Default is false."
+  default     = null
+}
+
+variable "bgp_best_path_selection_mode" {
```

Comment:
@YpNo if default value is standard can we set the default value to Standard instead of null?